coding-the-legislation/25_vectorial_computing.md

```diff
@@ -20,6 +20,33 @@ array([45, 42, 17])
 
 Basic operations such as `+` or `*` behave the same way on vectors than on numbers, you can thus use them in OpenFisca. However, some operations and structures must be adapted.
 
+## Scalars
+
+As a first-time developer, it is natural to write such a formula
```

Comment:
> This can read a bit condescending. What about “We are used to work with scalars rather than vectors. We thus have a tendency to write straightforward code that returns a scalar rather than a unidimensional vector:”